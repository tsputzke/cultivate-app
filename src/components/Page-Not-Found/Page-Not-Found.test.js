import React from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme'
import toJson from 'enzyme-to-json'
import PageNotFound from './Page-Not-Found'

configure({adapter: new Adapter()});  

describe(`PageNotFound component`, () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(
      <BrowserRouter>
        <PageNotFound />
      </BrowserRouter>,
      div
    )
    ReactDOM.unmountComponentAtNode(div)
  })

  it('renders PageNotFound by default', () => {
    const wrapper = shallow(<PageNotFound />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})