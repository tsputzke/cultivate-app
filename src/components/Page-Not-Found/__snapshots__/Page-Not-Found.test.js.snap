// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound component renders PageNotFound by default 1`] = `
<section
  className="center-align"
>
  <h1
    className="page-not-found-title title-style"
  >
    Error: Page does not exist.
  </h1>
</section>
`;
